### PR TITLE
6527 libvirt commands

### PR DESCRIPTION
**Pull request by h0st**

### CODE REVIEW

File: boxes.json [hunk 1]
```diff
@@ -179,6 +179,24 @@
...
+  "ubuntu_trusty64_libvirt": {
+    "provider": "libvirt",
+    "box": "baremettle/ubuntu-14.04",
+    "platform": "ubuntu",
+    "platform_version": "trusty"
+  },
+  "debian_7.5_libvirt": {
+    "provider": "libvirt",
+    "box": "baremettle/debian-7.5",
+    "platform": "ubuntu",
```
Comment (timofeyturenko) [inline boxes.json:197]: is it Debian or Ubuntu?

File: boxes.json [hunk 2]
```diff
@@ -179,6 +179,24 @@
     "platform": "rhel_ppc",
     "platform_version" : "7"
   },
+  "ubuntu_precise_libvirt": {
+    "provider": "libvirt",
+    "box": "bento/ubuntu-12.04",
+    "platform": "ubuntu",
+    "platform_version": "trusty"
```
Comment (timofeyturenko) [inline boxes.json:186]: is it precise or trusty?

**Comment by timofeyturenko** [comment]
shout "box": "uvsmtid/centos-7.0-minimal" be changed to "centos/7"?

File: boxes.json (hunk 2 above)
Comment (h0st) [inline boxes.json:186]: It is ubuntu precise.

File: boxes.json (hunk 1 above)
Comment (h0st) [inline boxes.json:197]: Yes, copy paste. It is debian 7.5.

**Comment by h0st** [comment]
Yes, should be changed to centos/7

File: README.md [hunk 3]
```diff
@@ -8,16 +8,17 @@
...
 
-Current version of mdbci is 0.4 (beta)
+Current version of mdbci is 0.6 (beta)
 
 ### Roadmap
 
 Following features are under development and will be available in the next versions of **mdbci**:
 
-* Support remote Linux systems as tager nodes
+* Support remote Linux systems as mdbci nodes
```
Comment (timofeyturenko) [inline README.md:17]: Please clean up unrelated stuff! (It does not have any connection to libvirt)

File: README.md [hunk 4]
```diff
@@ -309,7 +310,27 @@ MDBCI supports next VM providers:
 
 * VirtualBox 4.3 and upper
 * Amason EC2
-* _Remote PPC Boxes (under development)_
+* Remote PPC boxes (mdbci)
```
Comment (timofeyturenko) [inline README.md:313]: mdbci boxes feature is in 'integration' already. Please remove it from here

File: boxes.json [hunk 5]
```diff
@@ -179,6 +179,24 @@
     "platform": "rhel_ppc",
     "platform_version" : "7"
   },
+  "ubuntu_precise_libvirt": {
```
Comment (timofeyturenko) [inline boxes.json:182]: please check boxes.json in integration and remove unnecessary stuff from here

File: recipes/cookbooks/galera/recipes/install_galera.rb [hunk 6]
```diff
@@ -15,7 +15,11 @@
   when "rhel", "fedora", "centos"
     package "wget"
     if node[:platform] == "centos"
-      if node["platform_version"].to_f >= 6.0 
+      if node["platform_version"].to_f >= 7.0
+        execute "Install ifconfig" do
+          command "yum --assumeyes install net-tools"
```
Comment (timofeyturenko) [inline recipes/cookbooks/galera/recipes/install_galera.rb:20]: I doubt that this net-tools stuff is related to libvirt

File: recipes/cookbooks/mariadb-maxscale/recipes/install_maxscale.rb [hunk 7]
```diff
@@ -11,9 +11,19 @@
   end
 end  # Turn off SElinux
 
+# install ifconfig
+case node[:platform_family]
+  when "rhel", "centos"
+    if node[:platform] == "centos" and node["platform_version"].to_f >= 7.0
+      execute "Install ifconfig" do
+        command "yum --assumeyes install net-tools"
```
Comment (timofeyturenko) [inline recipes/cookbooks/mariadb-maxscale/recipes/install_maxscale.rb:19]: again net-tools! it is everywhere! in every pull request!

File: recipes/cookbooks/mariadb-maxscale/recipes/install_maxscale.rb [hunk 8]
```diff
@@ -11,9 +11,19 @@
...
+      end
+    end
+end
+
 # check and install iptables
 case node[:platform_family]
-  when "debian", "ubuntu"
+  when "debian", "ubuntu"
```
Comment (timofeyturenko) [inline recipes/cookbooks/mariadb-maxscale/recipes/install_maxscale.rb:26]: are two spaces really needed at the end of line ::)

File: recipes/cookbooks/mariadb/recipes/install_community.rb [hunk 9]
```diff
@@ -33,6 +33,16 @@
 
 system 'echo Platform family: '+node[:platform_family]
 
+# install ifconfig
+case node[:platform_family]
+  when "rhel", "centos"
+    if node[:platform] == "centos" and node["platform_version"].to_f >= 7.0
+      execute "Install ifconfig" do
+        command "yum --assumeyes install net-tools"
```
Comment (timofeyturenko) [inline recipes/cookbooks/mariadb/recipes/install_community.rb:41]: net-tools! again!

File: recipes/cookbooks/mariadb/recipes/install_community.rb [hunk 10]
```diff
@@ -98,16 +108,8 @@
     command "zypper -n install --from mariadb MariaDB-server MariaDB-client &> /vagrant/log"
   end
 when "debian"
-  #bash "MariaDB install" do
```
Comment (timofeyturenko) [inline recipes/cookbooks/mariadb/recipes/install_community.rb:101]: it looks like some debugging stuff
